modules/aerodyn/src/AeroDyn.f90

```diff
@@ -1031,7 +1031,7 @@ subroutine Init_u( u, p, p_AD, InputFileData, InitInp, errStat, errMsg )
       if (errStat >= AbortErrLev) return
             
          ! set node initial position/orientation
-      position = InitInp%HubPosition
+      position = InitInp%NacellePosition
       position(1:2) = 0
```

Comment:
This mesh was originally created to transfer nacelle drag loads, which are calculated at the tower center line at hub height with the orientation of the nacelle. If you want this to be the actual nacelle position, you should probably also remove the line setting position(1:2)=0.

Reply:
Thanks!  I'll do that.

Reply:
It would be great to add the drag loads at some point though. Should this mesh have two points then ? A nacelle center and an aerodynamic center?

Reply:
It seems like the aero center should be handled within AD15.  So I would expect a single mesh point is necessary for the loads to ED.

Reply:
Maybe two points on the mesh would be better.... I feel like we had a conversation with Hannah about this when I first sent her the code to add the nacelle mesh to AeroDyn. Did she alter her equations to be calculated at this different point???

Reply:
It looks like Hannah is referencing against the "nacelle reference point", which is given as the tower top in the documentation.  There are some offsets given somewhere center of mass offset that are then used for the resultant buoyancy forces: https://github.com/OpenFAST/openfast/pull/957/files#diff-dd7b98a4c7eb64ad7503601ab8789a1861273ede4896c0c161a447e05549d8b6R2037

Reply:
I do recall having a conversation with Jason and Hannah, discussing if we should have multiple points on that mesh or change the reference position, and I feel like they accounted for the difference in position in her equations in AD. But, I'd have to defer to her regarding what was actually implemented and pushed to OpenFAST.

Reply:
Just saw this. The buoyancy calculations for the nacelle assume a nacelle reference position aligned with the tower centerline and vertically with the hub, which is how the nacelle position was defined previously. I assume this means I need to adjust the buoyancy calculation now? Where is the new nacelle reference position? @andrew-platt

Reply:
The nacelle reference position from the ElastoDyn `NacelleLoads` and `NacelleMotion` mesh is located at `(0,0,TwrHt)`.  This is what is passed to AD15 for the Nacelle initial position.  So this should work as expected with the MHK buoyancy updates.

Reply:
Ok, I see. I think the key thing is to make sure it is clear in the documentation that the nacelle center of buoyancy input by the user should be relative to the tower top position. I believe Jason already suggested that "nacelle reference position" be changed to "yaw bearing", so that should take care of it.

We do have a calculation to move the correction for the tower top (since it is not exposed to fluid) from the center of buoyancy of the tower top node to the nacelle reference position. Since the tower center of buoyancy is assumed along the centerline and we are now defining the nacelle reference position as `(0,0,TwrHt)`, these points should be coincident, so I could remove this calculation. Does this sound correct, @jjonkman? I could also leave since it is more robust to future changes this way.

Reply:
@hkross -- Yes, that sounds correct, but may as well leave that code in there in case the nacelle reference position is moved again.

Reply:
I came across another potential issue with this. The AeroDyn driver does not pass the nacelle position to AeroDyn. Since the mesh creation in AeroDyn used to call `InitInp%HubPosition`, the nacelle position was not needed. This is fairly easy to fix and pass in the nacelle position. However, the nacelle position in the driver is set to `(0, 0, HubHt+overhang*sin(shftTilt))` rather than `(0,0,TwrHt)`, meaning that the nacelle position is defined differently when the AeroDyn driver vs OpenFAST driver is used. This will affect the buoyancy calculations, since the user inputs offsets for the nacelle center of buoyancy relative to the tower top. I'm not sure there is even enough info in the AeroDyn driver input file to be able to calculate `TwrHt`. @andrew-platt and @ebranlard, do either of you have any input on this?